sklearn/model_selection/tests/test_validation.py

```diff
@@ -2539,6 +2539,23 @@ def test_groups_with_routing_validation(func, extra_args):
         )
 
 
+@pytest.mark.parametrize(
+    "func, extra_args",
+    [
+        (cross_validate, {}),
+        (cross_val_score, {}),
+        (cross_val_predict, {}),
+        (learning_curve, {}),
+        (permutation_test_score, {}),
+        (validation_curve, {"param_name": "alpha", "param_range": np.array([1])}),
+    ],
+)
+@config_context(enable_metadata_routing=True)
+def test_cross_validate_params_none(func, extra_args):
```

Comment:
Can we add a docstring for the test? It would be difficult to understand what it's testing later otherwise.

Or maybe, we can join this test with the next one (`test_passed_unrequested_metadata`) and modify the docstring there.

Reply:
done